### DISCUSSION TITLE
Bad performance

### DISCUSSION THREAD

**Discussion by divdax** [dietercoopman/smart#22] [Q&A] [answered]
Hey,

i only have a simple route to load a view with `Route::view('/', 'index');`.
The **blank page loads** in **70ms**.

After i've added 20 images like this:
```html
<x-smart-image
  src="{{ resource_path('img/example[1-20].jpg') }}"
  data-template="sm"
  data-src="example[1-20].jpg"
/>
```
Now the **page loads** in **9s (9000ms)**. So this is suuuper slow.
Cache driver is set to file. Changing it has no effect. In the network-tab the route `/` is so slow. The served images are loaded from cache.

What i am missing?

**Comment by dietercoopman** [comment, marked as answer]
Hi @divdax , 

Thanks for reporting this issue, I've tried it with 20 images and it's indeed a problem ( because of the cache renaming - data-src ).  The good news is , I've fixed it.  The first render is still a slow one ( because of the first processing ) , but te second should be fast now.  My test got from 9s to 450ms. 

Could you please try it with version 1.5.2

Have a nice day !

**Comment by divdax** [comment]
@dietercoopman If i remember correctly i had the same issue even without `data-src`. Page load is now at 80ms after updating to v1.5.2. 🙏 🥳
